packages/compass-e2e-tests/tests/collection-explain-plan-tab.test.ts

```diff
@@ -21,87 +17,70 @@ describe('Collection explain plan tab', function () {
 
   beforeEach(async function () {
     await createNumbersCollection();
-    await browser.connectWithConnectionString(
-      `mongodb://localhost:27091/${dbName}`
-    );
-    await browser.navigateToCollectionTab(dbName, collectionName, tabName);
+    await browser.connectWithConnectionString();
+    newExplainPlanFeature = (await browser.getFeature(
+      'newExplainPlan'
+    )) as boolean;
```

Comment:
(feel free to ignore, this doesn’t seem to do anything when trying it out locally)

```suggestion
    ));
```

Reply:
Hmmm, I'm seeing this error locally without this:

```
Type 'string | number | boolean | [key: string, value: string][] | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.ts(2322)
 ```

Reply:
Hm yeah, not sure what causes that … the method should be typed properly, and it does work fine for me locally. But also, not super important, and this will go away anyway when the feature flag is removed, so :woman_shrugging:

Reply:
This is probably something with my machine, but just in case I'll leave it as, yeah, we are going to remove this pretty soon